content/software/plc-ide/tutorials/plc-programming-introduction/content.md

```diff
@@ -82,19 +82,19 @@ There are 4 Tasks:
 To assign a program to a task, you can easily drag and drop your program to the task.
 ![Drag and drop a program to task](assets/dragAndDrop_task.png)
 
-By default the `main` program is attached to the `Fast` task, you can remove it by right clicking and pressing the "remove program" option.
+By default the `main` program is attached to the `Fast` task, you can remove it by right-clicking and pressing the "remove program" option.
 
-You can also assign a new program by right clicking the task and pressing `Add program`, then select the program from the list shown in the pop up window.
+You can also assign a new program by right-clicking the task and pressing `Add program`, then select the program from the list shown in the pop-up window.
 
-If you want to configure the task, right click on it and press `Task configuration`, you will be able to, for example, change the cycle time of the task.
+If you want to configure the task, right-click on it and press `Task configuration`, you will be able to, for example, change the cycle time of the task.
 
 ***The execution order will be from top to bottom, the order shown inside each task.***
 
 ### Watch Variables
 
 To check that our program is running and counting correctly, we will use the **watch** feature which allows you to see the value of a variable in real time.
 
-To do so you need to go **View > Tool windows > Watch** and you will have a new tool window.
+To do so you need to go to **View**** > Tool** windows > Watch** and you will have a new tool window.
```

Comment:
What happened with the asteriskes here?

Reply:
Fixed, be careful with Grammarly sometimes it mess with *****